classesExamples/PassionFruit.py

```diff
@@ -0,0 +1,20 @@
+from classesExamples.Fruit import Fruit
+
```

Comment:
Is there a reason why there are always two line breaks after the import statements. I know it seems silly and annoying, but styling consistency is important, it will help you to make your code more readable, and to better align with teams standard when working with more people. Even when these are just for learning it will help you to get used to it.

Reply:
It's because of the language or the tool, not sure now, but If i don't do that, it throws me an error.

Reply:
fair enough